app/crud/metric.py

```diff
@@ -12,4 +12,20 @@ def __init__(self, db_session: Session):
 
     def getByCode(self, code: str):
         return self.db_session.query(Metric).filter(Metric.code == code).first()
+    
+    def getMetricType(self,metricId: int) -> str:
```

Comment:
this and `getMetricName` should be useless after applying the changes requested in the added comments, you can keep them or remove them it's up to you.